content/chapters/io/lab/solution/pipes/anonymous_pipe.c

```diff
@@ -9,14 +9,15 @@
 
 #include "utils/utils.h"
 
-#define BUFSIZE 	128
-#define EXIT_STR	"exit"
-#define PIPE_READ	0
-#define PIPE_WRITE	1
+#define BUFSIZE 128
+#define EXIT_STR "exit"
+#define PIPE_READ 0
+#define PIPE_WRITE 1
```

Comment:
The tabs were OK. It looks better to keep the `#define`s aligned.

Reply:
We should provide a `.clang-format` file so we could correct this automatically.

Reply:
`checkpatch.pl` can do this IIRC.